app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
 class WelcomeController < ApplicationController
-  def index; end
+  def index
+    @total_users = User.count
+    @total_questions = Question.count
+    @total_answers = Answer.count
+    @total_tenants = Tenant.count
```

Comment:
Remove, since total `tenants` was not specified in the requirements.

Reply:
https://github.com/SpicyCookies/rails-interview-project/pull/7/commits/084522578eb9a1fb1566174844436b319aa5008a#diff-0f1de9b0b214d551c91c7cf0b6b545c2c2cb7f8155d28cbb5db35e63fedf9946